dagfactory/dagbuilder.py

```diff
@@ -834,9 +839,8 @@ def configure_schedule(dag_params: Dict[str, Any], dag_kwargs: Dict[str, Any]) -
             is_airflow_version_at_least_2_4 = version.parse(AIRFLOW_VERSION) >= version.parse("2.4.0")
             is_airflow_version_at_least_2_9 = version.parse(AIRFLOW_VERSION) >= version.parse("2.9.0")
             has_schedule_attr = utils.check_dict_key(dag_params, "schedule")
-            has_schedule_interval_attr = utils.check_dict_key(dag_params, "schedule_interval")
 
-            if has_schedule_attr and not has_schedule_interval_attr and is_airflow_version_at_least_2_4:
+            if has_schedule_attr and is_airflow_version_at_least_2_4:
```

Comment:
The minimum version of Airflow is 2.4 in DAG-Factory now so we can remove check `is_airflow_version_at_least_2_4`